front/src/routes/integration/all/broadlink/peripheral-page/index.js

```diff
@@ -0,0 +1,23 @@
+import { Component } from 'preact';
+import { connect } from 'unistore/preact';
+import actions from './actions';
+import BroadlinkPage from '../BroadlinkPage';
+import PeripheralTab from './PeripheralTab';
+
+@connect('session,user,broadlinkPeripherals,getBroadlinkPeripheralsStatus,houses', actions)
```

Comment:
For now it's ok, but in the future, let's try to use less the global store (unistore) for this, especially here with global variables (houses), because it can create pollution around components

We had a bug for example when the `houses` global variable was filled with houses without rooms.

Maybe here, could you use a local call for get houses?